rag_experiment_accelerator/evaluation/plain_metrics.py

```diff
@@ -1,153 +1,163 @@
     """
-    score = int(algorithms.jaccard.normalized_similarity(value1, value2) * 100)
+    score = int(alg.jaccard.normalized_similarity(str1, str2) * 100)
     return score
 
 
-def hamming(value1, value2):
+def hamming(str1: str, str2: str) -> int:
     """
     Calculates the Hamming similarity score between two values.
 
     Args:
-        value1 (str): The first value to compare.
-        value2 (str): The second value to compare.
+        str1 (str): The first value to compare.
+        str2 (str): The second value to compare.
 
     Returns:
         int: The Hamming similarity score between the two values, as a percentage.
     """
-    score = int(algorithms.hamming.normalized_similarity(value1, value2) * 100)
+    score = int(d.Hamming.normalized_similarity(str1, str2) * 100)
     return score
 
 
-def jaro_winkler(value1, value2):
+def jaro_winkler(str1: str, str2: str) -> int:
     """
     Calculates the Jaro-Winkler similarity score between two strings.
 
     Args:
-        value1 (str): The first string to compare.
-        value2 (str): The second string to compare.
+        str1 (str): The first string to compare.
+        str2 (str): The second string to compare.
 
     Returns:
         int: The Jaro-Winkler similarity score between the two strings, as an integer between 0 and 100.
     """
-    score = int(algorithms.jaro_winkler.normalized_similarity(value1, value2) * 100)
+    score = int(d.JaroWinkler.normalized_similarity(str1, str2) * 100)
     return score
 
 
-def cosine(value1, value2):
+def cosine(str1: str, str2: str) -> int:
     """
     Calculates the cosine similarity (Ochiai coefficient) between two strings
     using token-frequency vectors
 
     https://en.wikipedia.org/wiki/Cosine_similarity.
 
     Args:
-        value1 (list): The first vector.
-        value2 (list): The second vector.
+        str1 (list): The first vector.
+        str2 (list): The second vector.
 
     Returns:
         int: The cosine similarity score between the two vectors, as a percentage.
     """
-    score = int(algorithms.cosine.normalized_similarity(value1, value2) * 100)
+    score = int(alg.cosine.normalized_similarity(str1, str2) * 100)
     return score
 
 
-def lcsseq(value1, value2):
+def lcsseq(str1: str, str2: str) -> int:
     """
     Computes the longest common subsequence (LCS) similarity score between two input strings.
 
     Args:
-        value1 (str): The first input string.
-        value2 (str): The second input string.
+        str1 (str): The first input string.
+        str2 (str): The second input string.
 
     Returns:
         int: The LCS similarity score between the two input strings, as a percentage (0-100).
     """
-    score = int(algorithms.lcsseq.normalized_similarity(value1, value2) * 100)
+    score = int(d.LCSseq.normalized_similarity(str1, str2) * 100)
     return score
 
 
-def lcsstr(value1, value2):
+def lcsstr(str1: str, str2: str) -> int:
     """
```

Comment:
Severity: Major - The `lcsstr` function's return type hint should be `float` instead of `int` to match the actual return type.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.